### ISSUE TITLE
Build error on ATTiny88 + ATTinyCore 1.5.2

### ISSUE THREAD

**Issue by eNovware** [nerdralph/debugSerial#3]
Cannot build my project for ATTiny88 (8MHz internal) with TX on pin PC1

looks like there are some #define  problems.

```
FQBN: ATTinyCore:avr:attinyx8:bod=2v7
Using board 'attinyx8' from platform in folder: C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2
Using core 'tiny' from platform in folder: C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2

Detecting libraries used...
C:\Users\####\AppData\Local\Arduino15\packages\arduino\tools\avr-gcc\7.3.0-atmel3.6.1-arduino7/bin/avr-g++ -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -flto -w -x c++ -E -CC -mmcu=attiny88 -DF_CPU=8000000L -DCLOCK_SOURCE=0 -DARDUINO=10607 -DARDUINO_AVR_ATTINYX8 -DARDUINO_ARCH_AVR -DNEOPIXELPORT=PORTA -IC:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny -IC:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\variants\tinyX8 C:\Users\####\AppData\Local\Temp\arduino\sketches\26988AED817C64171E9EDF3044746FB7\sketch\TEST.ino.cpp -o nul
Generating function prototypes...
C:\Users\####\AppData\Local\Arduino15\packages\arduino\tools\avr-gcc\7.3.0-atmel3.6.1-arduino7/bin/avr-g++ -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -flto -w -x c++ -E -CC -mmcu=attiny88 -DF_CPU=8000000L -DCLOCK_SOURCE=0 -DARDUINO=10607 -DARDUINO_AVR_ATTINYX8 -DARDUINO_ARCH_AVR -DNEOPIXELPORT=PORTA -IC:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny -IC:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\variants\tinyX8 C:\Users\####\AppData\Local\Temp\arduino\sketches\26988AED817C64171E9EDF3044746FB7\sketch\TEST.ino.cpp -o C:\Users\####\AppData\Local\Temp\1578754442\sketch_merged.cpp
C:\Users\####\AppData\Local\Arduino15\packages\builtin\tools\ctags\5.8-arduino11/ctags -u --language-force=c++ -f - --c++-kinds=svpf --fields=KSTtzns --line-directives C:\Users\####\AppData\Local\Temp\1578754442\sketch_merged.cpp
Compilation du croquis...
"C:\\Users\\####\\AppData\\Local\\Arduino15\\packages\\arduino\\tools\\avr-gcc\\7.3.0-atmel3.6.1-arduino7/bin/avr-g++" -c -g -Os -Wall -Wextra -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -MMD -flto -mmcu=attiny88 -DF_CPU=8000000L -DCLOCK_SOURCE=0 -DARDUINO=10607 -DARDUINO_AVR_ATTINYX8 -DARDUINO_ARCH_AVR -DNEOPIXELPORT=PORTA "-IC:\\Users\\####\\AppData\\Local\\Arduino15\\packages\\ATTinyCore\\hardware\\avr\\1.5.2\\cores\\tiny" "-IC:\\Users\\####\\AppData\\Local\\Arduino15\\packages\\ATTinyCore\\hardware\\avr\\1.5.2\\variants\\tinyX8" "C:\\Users\\####\\AppData\\Local\\Temp\\arduino\\sketches\\26988AED817C64171E9EDF3044746FB7\\sketch\\TEST.ino.cpp" -o "C:\\Users\\####\\AppData\\Local\\Temp\\arduino\\sketches\\26988AED817C64171E9EDF3044746FB7\\sketch\\TEST.ino.cpp.o"
In file included from D:\####\####\_####\####\Firmware\TEST\TEST.ino:3:0:
D:\####\####\_####\####\Firmware\TEST\debugSerial.h:23:0: warning: "bit" redefined
 #define bit(io)     BIT(io)
 
In file included from C:\Users\####\AppData\Local\Temp\arduino\sketches\26988AED817C64171E9EDF3044746FB7\sketch\TEST.ino.cpp:1:0:
C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny/Arduino.h:101:0: note: this is the location of the previous definition
 #define bit(b) (1UL << (b))
 
In file included from C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny/Stream.h:26:0,
                 from C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny/TinySoftwareSerial.h:5,
                 from C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny/Arduino.h:249,
                 from C:\Users\####\AppData\Local\Temp\arduino\sketches\26988AED817C64171E9EDF3044746FB7\sketch\TEST.ino.cpp:1:
C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny/Print.h:33:13: error: expected identifier before numeric constant
 #define DEC 10
             ^
D:\####\####\_####\####\Firmware\TEST\debugSerial.h:108:13: note: in expansion of macro 'DEC'
 enum _base {DEC = 10, HEX = 16};
             ^~~
C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny/Print.h:33:13: error: expected '}' before numeric constant
 #define DEC 10
             ^
D:\####\####\_####\####\Firmware\TEST\debugSerial.h:108:13: note: in expansion of macro 'DEC'
 enum _base {DEC = 10, HEX = 16};
             ^~~
C:\Users\####\AppData\Local\Arduino15\packages\ATTinyCore\hardware\avr\1.5.2\cores\tiny/Print.h:33:13: error: expected unqualified-id before numeric constant
 #define DEC 10
             ^
D:\####\####\_####\####\Firmware\TEST\debugSerial.h:108:13: note: in expansion of macro 'DEC'
 enum _base {DEC = 10, HEX = 16};
             ^~~
In file included from D:\####\####\_####\####\Firmware\TEST\TEST.ino:3:0:
D:\####\####\_####\####\Firmware\TEST\debugSerial.h:108:31: error: expected declaration before '}' token
 enum _base {DEC = 10, HEX = 16};
                               ^

exit status 1

Compilation error: expected declaration before '}' token
```

Any idea ?

I was hoping to replace the overkill SoftwareSerial library with your, I just wnat to log some events   :(

Thanks for the job.

**Comment by nerdralph** [comment]
ATTinyCore defines DEC & HEX differently than picoCore, which I was using to test debugSerial.  I'll change the definitions so their scope is limited.

**Comment by nerdralph** [comment]
fixed in commit ff3dfd6c0571949a437cc7f9230950a56ec05c57
